src/vos/evtree.c

```diff
@@ -2568,6 +2568,14 @@ evt_ent_array_fill(struct evt_context *tcx, enum evt_find_opc find_opc,
 				if (rc < 0)
 					D_GOTO(out, rc);
 
+				if (range_overlap == RT_OVERLAP_INCLUDED &&
```

Comment:
is this still needed?  It should be safe but Di says his patch avoids partial overwrite issue

Reply:
@wangdi1 's patch only for rebuild case that will reclaim rebuilt obj when rebuild failed.
but here is for another case of minor epoch -1, that is not for rebuild, but it is for vos_obj_array_remove(). I think Di's patch cannot avoid this case.
a few detailed log in DAOS-9785, here just don't return err but keep an err log.